Graphs/DynamicBarGraph/barGraph.test.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import {CheckTicks,getDataAndConfig} from '../testHelper';
+import BarGraph from '.'; 
+const cheerio = require('cheerio')
+
+describe ("test barGrpah", ()=>{
+    beforeAll( async () => {
+            global.config = await getDataAndConfig('DynamicBarGraph');     
+    });
+
+    describe("positive-data-horizontal-number", ()=>{
+        
+        beforeAll( async () => {
+            global.componenthorizontal = mount(<BarGraph width={500} height={500} configuration={config.horizontal} data={config.data}> </BarGraph>);
+        });
+
+        it("number of barGraph is matched", ()=>{
+            const middleHtml = componenthorizontal.find('.graph-bars').html();
+            const $ = cheerio.load(middleHtml);
+            const noOfBars = $('.graph-bars').find('g').length;
+            const totalBar = global.componenthorizontal.root.props().configuration.data.otherOptions.limit + 1;
+            expect(noOfBars).toBe(totalBar)
+        });
+
+        it("check xAxis ticks", ()=>{
+            const xAxisTicks = CheckTicks(componenthorizontal,'.graph-container','.xAxis','g')
+            expect(xAxisTicks).toBeGreaterThan(0);
+        });
+    
+        it("check yAxis ticks", ()=>{
+            const yAxisTicks = CheckTicks(componenthorizontal,'.graph-container','.yAxis','g')
+            expect(yAxisTicks).toBeGreaterThan(0);
+        });
+        
+    });   
+
+    describe("vertical-percentage", ()=>{
+        beforeAll( async () => {
+            global.componentVertical = mount(<BarGraph width={500} height={500} configuration={config.vertical} data={config.data}> </BarGraph>); 
```

Comment:
Remove global from all the areas and files and use ` this.verticalBar = ..... `

Reply:
Also please take care of the lin length. Please split it into multiple lines.

Reply:
Done